### ISSUE TITLE
Crashes Homebridge

### ISSUE THREAD

**Issue by robertherbaugh** [naofireblade/homebridge-neato#73]
I just installed this plugin this morning on my home bridge container. Immediately it starts utilizing all system resources and brings down the entire docker server.
<img width="678" alt="Screen Shot 2022-01-10 at 11 52 21" src="https://user-images.githubusercontent.com/63149639/148805502-6113e307-cf5f-417e-aab1-e1a7e5ec7ba6.png">

**Comment by VariantT505** [comment]
Yes, it uses all ressources and slows down homebridge. There is a warning in the logs every hour. Sadly, it doesn't seem like this plugin is maintained anymore.
